### ISSUE TITLE
VTAB-1k dataset corruption with JPEG

### ISSUE THREAD

**Issue by yuan-yin** [Davidzhangyuanhan/NOAH#28]
Hello,

Just a note: the VTAB-1k datasets were extracted from the original train, val and test sets using a lossy JPEG format corrupting the data, which may affect fine-tuning performance for some models. For example, for CLIP/SIGLIP, even performing zero-shot classification on the provided CIFAR100 test set shows a 20 % drop in accuracy.
